test/integration/admin_dashboard_test.rb

```diff
@@ -11,6 +11,18 @@ class AdminDashboardTest < ActionDispatch::IntegrationTest
 
     assert page.has_content?('Welcome, Admin!')
   end
+  test 'when admin logs they are redirected to dashboard page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.